theano/configdefaults.py

```diff
@@ -418,6 +419,19 @@ def f(suffix=suffix):
 except OSError:
     rc = 1
 
+# Anaconda on Windows has mingw-w64 packages including GCC, but it may not be on PATH.
+if rc != 0:
+    if sys.platform == "win32":
+        mingw_w64_gcc = os.path.join(os.path.dirname(sys.executable), "Library", "mingw-w64", "bin", "g++")
```

Comment:
Yes. That supports both no problem.

mingw-w64 is just the project name to differentiate it from mingw.org.